recipes/txaio/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "txaio" %}
+{% set version = "2.5.2" %}
+{% set sha256 = "321d441b336447b72dbe81a4d73470414454baf0543ec701fcfecbf4dcbda0fe" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - six
```

Comment:
It doesn't look like it, so I'll clean that up in the next commit. Any thoughts on how to handle the "extra" dependencies?